pom.xml

```diff
@@ -457,7 +457,7 @@
             <dependency>
                 <groupId>com.clickhouse</groupId>
                 <artifactId>clickhouse-jdbc</artifactId>
-                <version>0.5.0</version>
```

Comment:
Are there any compatibility changes with this update?

Reply:
We test against:

- 21.8.14.5 (EOL 21 Aug 2022)
- 21.11.10.1
- 21.8.13.1.altinitystable
- 20.8.4.11_aes

Coverage didn't change

Reply:
Does it answer your concern @martint ?

Reply:
Yup, thanks.